src/ksync/ksync_sock.cc

```diff
@@ -249,25 +261,23 @@ KSyncSock::~KSyncSock() {
         free(nl_client_->cl_buf);
     }
     free(nl_client_);
-#endif
 }
 
 void KSyncSock::Shutdown() {
-#ifndef _WINDOWS //WINDOWSFIX
-
     shutdown_ = true;
     sock_->send_queue_.Shutdown();
     sock_.release();
-#endif
 }
 
 void KSyncSock::Init(bool use_work_queue) {
-#ifndef _WINDOWS //WINDOWSFIX
-
     sock_->send_queue_.Init(use_work_queue);
+    // TODO: Use some more generalised version
+#ifndef _WINDOWS
     pid_ = getpid();
-    shutdown_ = false;
+#else
+    pid_ = GetCurrentProcessId();
```

Comment:
windows_getpid is defined in Windows Stubs (it calls _getpid, which supposedly calls GetCurrentProcessId). Shall we use windows_getpid for consistency?